packages/integration-karma/test/api/isNodeFromTemplate/index.spec.js

```diff
@@ -33,33 +33,35 @@ it('should return false on the shadow root', () => {
     expect(isNodeFromTemplate(elm.shadowRoot)).toBe(false);
 });
 
-it('should return true on elements rendered from the template', () => {
-    const elm = createElement('x-test', { is: Test });
-    document.body.appendChild(elm);
+if (!process.env.MIXED_SHADOW) {
```

Comment:
As a side note. This will certainly cause current Locker to fail in mixed shadow mode. Locker uses `isNodeFromTemplate` to know if a given DOM node should be lockerized or not.